crl-janitor/README.md

```diff
@@ -52,5 +52,6 @@ No requirements.
 | cloudsql\_instance\_name | CRL Janitor CloudSQL instance name |
 | cloudsql\_root\_user\_password | CRL Janitor database root password |
 | cloudsql\_app\_db\_creds | CRL Janitor database user credentials |
-| cloudsql\_app\_stairway\_db\_creds | CRL Janitor Stairway database user credentials |
-
+| project\_id\ | CRL Janitor Google Project Id |
```

Comment:
terraform style question: is having an output that's the same as an input valuable?

Reply:
https://github.com/broadinstitute/terraform-ap-deployments/blob/master/terra-env/vault_crl_janitor.tf#L8
I think we don't need the project_id as input.

Reply:
ah, sorry, added it back